onnxruntime/core/providers/cpu/nn/string_normalizer.cc

```diff
@@ -201,7 +201,15 @@ class Utf8Converter {
 
 #endif
 
-const std::string default_locale("en_US.UTF-8");  // All non-MS
+#if defined(__APPLE__)
+#if TARGET_OS_IPHONE
```

Comment:
I believe this requires `#include <TargetConditionals.h>`.

for example,
https://github.com/microsoft/onnxruntime/blob/c6b0d185b47e2b4ad48425a2302383915b2a9f03/onnxruntime/test/unittest_main/test_main.cc#L50-L56

Reply:
ah okay thanks! updated.